ext/statistics/theme.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shimmie2;
+
+use MicroHTML\HTMLElement;
+
+use function MicroHTML\{TABLE,THEAD,TBODY,TR,TH,TD};
+use function MicroHTML\A;
+use function MicroHTML\B;
+use function MicroHTML\P;
+use function MicroHTML\DIV;
+use function MicroHTML\emptyHTML;
+use function MicroHTML\rawHTML;
+
+class StatisticsTheme extends Themelet
+{
+    public function display_page(Page $page, int $limit, ?HTMLElement $tag_table, ?HTMLElement $upload_table, ?HTMLElement $comment_table, ?HTMLElement $favorite_table): void
+    {
+        $html = emptyHTML();
+
+        if ($tag_table) {
+            $html->appendChild($tag_table);
+        }
+        if ($upload_table) {
+            $html->appendChild($upload_table);
+        }
+        if ($comment_table) {
+            $html->appendChild($comment_table);
+        }
+        if ($favorite_table) {
+            $html->appendChild($favorite_table);
+        }
+
+        $page->set_title(html_escape("Stats"));
+        $page->set_heading(html_escape("Stats - Top $limit"));
+        $page->add_block(new NavBlock());
+        $page->add_block(new Block("Stats", $html, "main", 20));
+    }
```

Comment:
MicroHTML itself checks for and ignores `null` elements, so this could be simplified to something like
```
$html = emptyHTML(
    $tag_table,
    $upload_table,
    $comment_table,
    $favorite_table,
);
```